spec/features/invite_spec.rb

```diff
@@ -0,0 +1,59 @@
+require 'spec_helper'
+
+feature 'invitation' do
+
+  given(:tom) { Fabricate(:user, full_name: 'Tom, the cat') }
+  given(:jerrys_email) { 'jerry@mice.com' }
+  given(:jerrys_name) { 'Jerry, the mouse' }
+
+  background 'tom is signed-in and invites jerry' do
+
+    clear_emails
+    sign_in tom
+
+    visit invite_path
+    p page.body
+    fill_in :invitation_email, with: jerrys_email
+    fill_in :invitation_full_name, with: jerrys_name
+    click_button 'Send Invitation'
+    open_email jerrys_email
+  end
+
+  scenario 'jerry receives an email' do
+    expect(current_email.to).to include jerrys_email
+    expect(current_email.body).to have_content jerrys_name
+  end
+
+  scenario 'jerry can confirm the invitation' do
+    # workaround: replace the host:port
+    href = confirm_invitation_url(Invitation.last.token, host: 'localhost:3000')
```

Comment:
okay; I fixed that. It was actually stupid to build the url instead of the path.